cmd/eksctl-anywhere/cmd/import_images.go

```diff
@@ -48,6 +48,7 @@ func init() {
 		log.Fatalf("Cannot mark 'bundles' as required: %s", err)
 	}
 	importImagesCmd.Flags().BoolVar(&importImagesCommand.includePackages, "include-packages", false, "Flag to indicate inclusion of curated packages in imported images")
+	importImagesCmd.Flag("include-packages").Deprecated = "Use copy packages command"
```

Comment:
```suggestion
	importImagesCmd.Flag("include-packages").Deprecated = "use copy packages command"
```
Just for consistency with the other deprecation on line 45 of downloadimages.go